index.js

```diff
@@ -0,0 +1,13 @@
+let divide = (a, b) => {
```

Comment:
`let`s will for sure work here, but even better would be `const`s. Anything that's fixed can and should be a `const`.